view/theme/frio/templates/sub/punct_wrap.tpl

```diff
@@ -0,0 +1 @@
+{{$text|regex_replace:"/([@\.])/":"<wbr>$1" nofilter}}

```

Comment:
Shouldn't it be `<wbr />` according to HTML standards?

Reply:
We currently mix converting text through `BBCode` (and similar) PHP classes and now with the template engine. Shouldn't we stay on one way to alter text? To improve maintainability?

Reply:
> Shouldn't it be `<wbr />` according to HTML standards?

According to `XHTML` standards, yes, but HTML 5 went back on them, including the need to close empty tags.

> Shouldn't we stay on one way to alter text? To improve maintainability?

Not necessarily, we only need to add punctuation wrappers to specific content in specific locations depending on the template, so this wouldn't make sense in the general BBCode conversion routine.